### PR TITLE
Update ewon.js to use ajax.

### PR DESCRIPTION
**Pull request by joseanor**

### CODE REVIEW

File: ABDemo/js/ewon.js [hunk 1]
```diff
@@ -31,6 +31,26 @@ $("*").click(function (event) {
 	}
 });
 
+
+//retrieve the lamp_status value on boot.
```
Comment (chmorgan) [inline ABDemo/js/ewon.js]: this comment doesn't look to match the function. It looks like it is used to set the value.

**Comment by chmorgan** [comment]
I'd probably also use the body of your commit message as the first line, the subject, as it explains what you are changing. If you don't need a body to the commit message to explain anything else you can leave it out.

File: ABDemo/js/ewon.js (hunk 1 above)
Comment (joseanor) [inline ABDemo/js/ewon.js]: @chmorgan You are completely correct. I did not erase that comment after removing
the previous GETTER and replacing it with a pseudo getter/setter.

**Comment by joseanor** [comment]
Updated initial commit with desired changes.

File: ABDemo/js/ewon.js [hunk 2]
```diff
@@ -31,6 +31,26 @@ $("*").click(function (event) {
 	}
 });
 
+
+//Set the lamp_status value on boot and updates throughout program run.
```
Comment (cmo-hms) [inline ABDemo/js/ewon.js]: I'd probably shorten this to "Set the lamp_status value." Imo you have no way of knowing who or how this function is being used, and if someone wanted to know that they could search for its usage in the rest of the code. It would be tough to keep updating the comment with all of the ways it is being used if the code got more complex.

File: ABDemo/index.shtm [hunk 3]
```diff
@@ -15,7 +15,7 @@
 	</head>
 	
 	
-	<body>
+	<body onload="SetLampValue(<%#TagSSI,Lamp_Status%>);">
```
Comment (cmo-hms) [inline ABDemo/index.shtm:18]: I can't comment on the commit log but I'd probably shorten it to a subject of "Update ewon.js to use ajax." and get rid of the subject line you are using now, "Update functions"

**Comment by cmo-hms** [comment]
+1 from me. Did @dbe-hms review this PR and approve it already?

**Comment by dbe-hms** [comment]
+1 looks good

**Comment by joseanor** [comment]
@cmo-hms good to merge?

**Comment by cmo-hms** [comment]
@jose-hms yep, two approvals looks good!
